cache.ts

```diff
@@ -21,6 +21,13 @@ export class FetchCacher {
     checksum?: string,
   ): Promise<LoadResponse | undefined> => {
     const url = new URL(specifier);
-    return this.#fileFetcher.fetchOnce(url, { cacheSetting, checksum });
+    return this.#fileFetcher.fetchOnce(url, { cacheSetting, checksum })
+      .catch((e) => {
+        if (e instanceof Deno.errors.NotFound) {
```

Comment:
Should we move this code down into fileFetcher.fetchOnce instead? It looks like it's already handled for 404. Actually, it looks like the code is handled in fetchLocal too.

Reply:
I put this here, because in Deno CLI the same kind of error handling is performed in `load` method, instead of in file_fetcher. See https://github.com/denoland/deno/blob/1a0cb5b5312941521ab021cfe9eaed498f35900b/cli/cache/mod.rs#L372-L426, which is `Loader` trait impl for `FetchCacher`, where `file_fetcher.fetch_no_follow_with_options` is called, followed by the error handling in `unwrap_or_else` checking if the error is `NotFound`.
So I think that putting `e instanceof Deno.errors.NotFound` here would be more consistent.